guides/common/modules/con_content-sources-in-project.adoc

```diff
@@ -0,0 +1,14 @@
+[id="content-sources-in-{ProjectNameID}_{context}"]
+= Content sources in {Project}
+
+The Red{nbsp}Hat subscription manifest determines what Red{nbsp}Hat repositories are accessible from your {ProjectServer}.
+Once you enable a Red{nbsp}Hat repository, an associated {Project} product is created automatically.
+For distributing content from custom sources you need to create products and repositories manually.
+Red{nbsp}Hat repositories are signed with GPG keys by default, and it is recommended to create GPG keys also for your custom repositories.
+
+Yum repositories that contain only RPM packages support the *On demand* download policy, which reduces synchronization time and storage space.
+The *On demand* download policy saves space and time by only downloading packages when requested by hosts.
+For detailed instructions on setting up content sources, see {ContentManagementDocURL}Importing_Content_content-management[Importing Content] in _{ContentManagementDocTitle}_.
+
+A custom repository within {ProjectServer} is in most cases populated with content from an external staging server.
```

Comment:
"in most cases" < this is probably Satellite-specific because for everyone else, custom content is just content.